docs/.custom_wordlist.txt

```diff
@@ -75,6 +75,7 @@ LTS
 MacOS
 macOS
 manpages
+Mantic
```

Comment:
it can be removed once the version of the release is updated to use the LTS

Reply:
I amended this changed and removed it from the wordlist after updating references to the Ubuntu LTS.